internal/repository/master_index.go

```diff
@@ -237,6 +237,27 @@ func (mi *MasterIndex) Each(ctx context.Context) <-chan restic.PackedBlob {
 	return ch
 }
 
+func (mi *MasterIndex) MergeFinalIndexes() {
```

Comment:
Nor this.

Reply:
I added tests which use this method.

Reply:
I usually add tests in the same package when testing something that isn't supposed to be called from another package. That way, you don't have to grep the entire repo to find the clients of a method before changing it.

It's a not a big deal, though.